ecosystem/indexer-grpc/indexer-grpc-table-info/src/table_info_parser.rs

```diff
@@ -93,7 +92,27 @@ impl TableInfoParser {
             tasks.push(task);
         }
         match futures::future::try_join_all(tasks).await {
-            Ok(res) => res,
+            Ok(res) => {
+                if !db.reader.is_indexer_async_v2_pending_on_empty().unwrap() {
```

Comment:
curious why the unwrap is needed. I thought that it returns a bool?

Reply:
`is_indexer_async_v2_pending_on_empty` returns <Result> cuz we wrap it around with `gauged_api` which comes with the benefit of recording the latency metrics for aptosdb api